### PR TITLE
Also test predispatch serialization

### PR DESCRIPTION
**Pull request by tugsbayasgalan**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #121687
* #121678
* __->__ #121652

### CODE REVIEW

**Review by zhxchen17 — APPROVED** [review]

File: test/export/test_serialize.py [hunk 1]
```diff
@@ -233,105 +233,114 @@ def forward(self, x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
...
+                        self.assertTrue(torch.allclose(orig, loaded))
+                else:
+                    self.assertEqual(orig, loaded)
+            self._check_graph_nodes(ep.graph_module, deserialized_ep.graph_module, _check_meta)
+
+        if use_pre_dispatch:
+            _check_graph(pre_dispatch=True)
+            _check_graph(pre_dispatch=False)
+        else:
+            _check_graph(pre_dispatch=False)
```
Comment (angelayi) [inline test/export/test_serialize.py:343]:
```suggestion
        if use_pre_dispatch:
            _check_graph(pre_dispatch=True)
        _check_graph(pre_dispatch=False)
```

**Review by angelayi — APPROVED** [review]

File: test/export/test_serialize.py (hunk 1 above)
Comment (angelayi) [reply test/export/test_serialize.py:343]: yeah -- the `_check_graph(pre_dispatch=False)` is outside of the if statement :P

Comment (tugsbayasgalan) [reply test/export/test_serialize.py:343]: Oh hahaha

**Pull request closed without merging**